Components/Pages/HomeTab.js

```diff
@@ -21,13 +21,11 @@ const HomeTabButtonInactive = ({ info }) => {
       style={{
         alignItems: "center",
         justifyContent: "center",
-        shadowRadius: 10,
         borderRadius: 12,
-        backgroundColor: "#DDDDDD",
         width: 50,
         height: 50,
```

Comment:
I think there is a bit too much padding, which maybe you can handle by making the buttons a bit bigger and/or making padding a bit smaller
![Screenshot 2020-07-12 13 36 54](https://user-images.githubusercontent.com/34425825/87253036-df845280-c445-11ea-8fa5-8e6d5c492a48.png)

Reply:
yeah these margins were tough the make it look good across devices. It is centered on latest iPhones, but no so much on older ones.

I am gonna leave it as is. If it becomes an issue down the road, feel free to bring it up again and create a bug for it.